cmake/support_catalyst.cmake

```diff
@@ -42,7 +44,22 @@ macro(FindCatalyst target_name)
                 ${HEADER_NAME}
                 URL                 https://raw.githubusercontent.com/PennyLaneAI/catalyst/${CATALYST_GIT_TAG}/runtime/lib/backend/common/${HEADER}
                 DOWNLOAD_NO_EXTRACT True
-                SOURCE_DIR          include
+                SOURCE_DIR          ../../include
+            )
+
+            FetchContent_MakeAvailable(${HEADER_NAME})
+        endforeach()
+
+        # Fetching /lib/capi hpp headers
```

Comment:
Could you elaborate on this dependency? Ideally, this shouldn't be the case for any runtime devices.

Reply:
Removed